src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Tools to measure Core Web Vitals"
+subhead: Your favorite developer tools can now measure the Core Web Vitals metrics.
+authors:
+  - addyosmani
+  - egsweeny
+date: 2020-05-28
+description: |
+  Read about the newly announced Core Web Vitals measurement support across popular web developer tools like Lighthouse, PageSpeed Insights, Chrome UX Report, and many others.
+hero: hero.png
+thumbnail: thumbnail.png
+alt: Chrome User Experience logo, PageSpeed Insights logo, Lighthouse logo, Search Console logo, Chrome DevTools logo, Web Vitals extension logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - web-vitals
+  - performance
+---
+
+The recently announced [Web Vitals](/vitals/) initiative provides unified guidance about quality signals that are essential for all sites to deliver a great user experience on the web. We're happy to announce that **all of Google's popular tools for web developers now support measurement of Core Web Vitals**, helping you more easily diagnose and fix user experience issues. This includes [Lighthouse](https://github.com/GoogleChrome/lighthouse), [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/), [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools), [Search Console](https://search.google.com/search-console/about), [web.dev's measure tool](/measure/), the [Web Vitals Chrome extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma) and a new (!) [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report) API. 
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./Vitals-Tools1.png" alt="Summary of Chrome and Search Tools that support the Core Web Vitals metrics">
+</figure>
+
+**_[Quick Refresher]_** Lab tools provide insight into how a _potential user_ will likely experience your website and offer reproducible results for debugging. **Field** tools provide insight into how your _real users_ are experiencing your website; this type of measurement is often called Real User Monitoring (RUM). Each [lab or field tool](/how-to-measure-speed/#lab-data-vs-field-data) offers distinct value for optimizing your user experience. 
+
+To begin your journey optimizing user-experience with Core Web Vitals, try the following workflow: 
```

Comment:
```suggestion
{% Aside 'key-term' %}
Lab tools provide insight into how a _potential user_ will likely experience your website and offer reproducible results for debugging. 
{% endAside %}

{% Aside 'key-term' %}
Field tools provide insight into how your _real users_ are experiencing your website; this type of measurement is often called Real User Monitoring (RUM). 
{% endAside %}

Each [lab or field tool](/how-to-measure-speed/#lab-data-vs-field-data) offers distinct value for optimizing your user experience. To begin your journey optimizing user-experience with Core Web Vitals, try the following workflow: 
```

Reply:
"Quick refresher" seems arbitrary, I think key term aside formatting would serve well here :) https://web.dev/handbook/web-dev-components/#key-term-asides

Reply:
Thanks! I'll talk to Elizabeth about us switching over to the key term formatting before we land this :)